src/frontend/src/views/Upload/components/Metadata/components/ImportFile/parseFile.ts

```diff
@@ -89,9 +89,17 @@ function getMissingHeaderFields(
   )) {
     if (
       !uploadedHeaders.includes(metadataKey) &&
-      !headerField.includes("Optional")
+      !headerField.includes("Optional") &&
+      metadataKey !== "keepPrivate" // TODO: rename field to have -Optional flag
```

Comment:
What is stopping us from making the "keepPrivate" key optional?

Reply:
it's a larger product/design decision. Mainly we don't want users to have to use a different template if it's not really a breaking change